### DISCUSSION TITLE
Easy way to get the whole data set?

### DISCUSSION THREAD

**Discussion by l3onh4rd** [theOehrly/Fast-F1#635] [Q&A]
Is there any easy way to get the whole data from a season or a grand prix weekend?

**Comment by Casper-Guo** [comment]
Define whole data? What format do you want the output in? You should be able to just use a for loop and iterate over all the sessions you are interested in and load the relevant data

**Reply by l3onh4rd** [reply]
Yeah sure, I know that this is a possible way. Before I start with that I just wanted to know if there is an easy way to get just every data point. Maybe like an csv file or many csv files.

Or even better I get to know what this library uses as source for the data. Maybe I can download it there as a whole data set.
